client/client.go

```diff
@@ -242,6 +243,50 @@ func (c Client) DestroyInstance(instance models.Instance) error {
 	return nil
 }
 
+type createImageRequest struct {
```

Comment:
Thoughts on making `routes.createImageRequest` and `routes.createInstanceRequest` public and then referring to them here?